autocti/charge_injection/imaging/imaging.py

```diff
@@ -43,6 +44,22 @@ def __init__(
 
         self.layout = layout
 
+        if fpr_value is None:
+
+            fpr_value = np.round(
+                np.mean(
+                    self.layout.extract.parallel_fpr.median_list_from(
+                        array=self.data,
+                        pixels_from_end=min(
+                            10, self.layout.smallest_parallel_rows_within_ci_regions

```

Comment:
Maybe assign 10 to a constant so it's clear what it means rather than just being an arbitrary integer value?